roles/sap_netweaver_preconfigure/tasks/sapnote/assert-3119751.yml

```diff
@@ -5,30 +5,46 @@
     msg: "SAP note {{ (__sap_netweaver_preconfigure_sapnotes_versions | selectattr('number', 'match', '^3119751$') | first).number }}
           (version {{ (__sap_netweaver_preconfigure_sapnotes_versions | selectattr('number', 'match', '^3119751$') | first).version }}): Linux Requirements for SAP Kernel 754 and for SAP Kernel 788 and higher"
 
+- name: Get info about the compat-sap-c++-10.so file
+  ansible.builtin.stat:
+    path: /opt/rh/SAP/lib64/compat-sap-c++-10.so
+  register: __sap_netweaver_preconfigure_register_stat_compat_sap_cpp
+
+- name: Report if checking for a link from libstdc++.so.6 to compat-sap-c++-10.so is skipped
+  ansible.builtin.debug:
+    msg: "INFO: Not checking for link {{ sap_netweaver_preconfigure_rpath }}/libstdc++.so.6 - file /opt/rh/SAP/lib64/compat-sap-c++-10.so does not exist on this system."
+  when: not __sap_netweaver_preconfigure_register_stat_compat_sap_cpp.stat.exists
+
 - name: Get info about file "{{ sap_netweaver_preconfigure_rpath }}/libstdc++.so.6"
```

Comment:
Suggesting improvement:
Use quotes around the entire task name, do not start in the middle of the name.